zerver/views/auth.py

```diff
@@ -492,6 +492,9 @@ def oauth_redirect_to_root(
     is_signup: bool = False,
     extra_url_params: Dict[str, str] = {},
     next: Optional[str] = REQ(default=None),
+    multiuse_object_key: str = REQ(default=""),
```

Comment:
Do we want to use a `check_string` here?

Reply:
I think `check_string` is not necessary here, and the same for the ones below. See also: #18035.

Reply:
Yeah, `check_string` seems redundant in these situations, we don't use it anywhere except the one strange place in `update_user_backend` tracked in #18409

Reply:
We can use `str_validator` if we want to check details of the string, but yeah, we shouldn't use a `json_validator` for a string.